Cargo.toml

```diff
@@ -68,6 +68,8 @@ base64 = "0.13"
 wait-timeout = "0.2"
 
 notify = "5.0.0-pre.7"
+strum_macros = "0.20.1"
+strum = "0.20.0"
```

Comment:
Not a big fan of adding these, writing that tiny bit of `match self` yourself should be fine - no need for additional macro compile time overhead and the inherent issues with generating user-facing config from identifiers in code in a non-obvious way